go/Makefile

```
@@ -1,4 +1,4 @@
-.PHONY: install fmt test-unit test-full build generate lint pull up down
+.PHONY: install fmt test-unit test-integration test-integration-long test-stress build generate lint pull up down
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)